### PR TITLE
feat(atomic): changed appearance of quickview button

### PR DESCRIPTION
**Pull request by btaillon**
https://coveord.atlassian.net/browse/KIT-886

On a compact result:
image

On a comfortable result:
image

### CODE REVIEW

File: packages/atomic/src/components/result-template-components-v1/atomic-result-quickview/atomic-result-quickview.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
+import {Component, State, h, Element, Prop} from '@stencil/core';
```
Comment (btaillon) [inline packages/atomic/src/components/result-template-components-v1/atomic-result-quickview/atomic-result-quickview.tsx:1]: [Diff (external link)](https://gist.github.com/btaillon/cab6ceec3b89e599c463fe749b52a93b/revisions#diff-3b0ecccb619f0cb9acc3ef0d88a903e4d5573507c9ffb6e4bdec1449016e34e7)

File: packages/atomic/src/components/result-template-components-v1/atomic-result-quickview/atomic-result-quickview.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    if (!this.quickviewState.resultHasPreview) {
+      return this.removeComponent();
+    }
+
+    const button = (
+      <button
+        class="btn-outline-primary grid h-full place-items-center"
+        aria-label={this.strings.previewResult()}
+        onClick={() => this.openModal()}
+        onMouseDown={(e) => createRipple(e, {color: 'neutral'})}
```
Comment (ThibodeauJF) [inline packages/atomic/src/components/result-template-components-v1/atomic-result-quickview/atomic-result-quickview.tsx:97]: 🔥  🚒

File: packages/atomic/src/components/result-template-components-v1/atomic-result-quickview/atomic-result-quickview.tsx (hunk 1 above)
Comment (ThibodeauJF) [reply packages/atomic/src/components/result-template-components-v1/atomic-result-quickview/atomic-result-quickview.tsx:1]: Previous component was basically unused as it was not a finished product, could we just overwrite the original for this one?

**Review by samisayegh — APPROVED** [review]

**Review by nbuissoncoveo — APPROVED** [review]
💯

**Pull request merged**